ui/apps/notifications/src/components/pages/customize-notification-page.tsx

```diff
@@ -178,18 +178,17 @@ const CustomizeNotificationPage: React.FC<CustomizeNotificationPageProps> = ({
     setLoading(true);
 
     try {
-      const allPrefs = allStates;
+      const allPrefs = structuredClone(allStates);
 
       //add snooze pref
       allPrefs[SnoozeOption] = snoozed;
 
       saveSettingsMutation.mutate(
```

Comment:
use-settings hooks still using react-query

Reply:
Hi Sever! I think I removed react-query from it already :) https://github.com/AKASHAorg/akasha-core/pull/2200/files#diff-b427a501346a4e029682a4ca0c6767651953ab2ca72c6e20966daed1cf228490